plone/app/dexterity/behaviors/tests/test_metadata.py

```diff
@@ -65,6 +70,35 @@ def test_subjects_getter(self):
         self.assertEqual((u'føø',), b.subjects)
 
 
+class CategorizationIntegrationTests(unittest.TestCase):
+    layer = DEXTERITY_INTEGRATION_TESTING
+
+    def test_language(self):
+        """See, what values are available in the vocabulary of the language
+        field. It should reflect, what's configured in the language control
+        panel.
+        """
+        # set available languages
+        registry = getUtility(IRegistry)
+        registry['plone.available_languages'] = ['en', 'de', 'hu']
+
+        field = ICategorization._InterfaceClass__attrs['language']
```

Comment:
Not so happy with the use of an "internal" attribute. However, thats the cheapest way.